Mobile/starter_project_matador/pubspec.yaml

```diff
@@ -59,9 +59,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
```

Comment:
check out the comment about this on https://github.com/RealEskalate/Internship-2023/pull/30/files